frontend/packages/client/src/components/Community/StrategyEditorModal/StrategyInformationForm.js

```diff
@@ -25,6 +26,15 @@ export default function StrategyInformationForm({
     resolver: yupResolver(getSchema(formFields, isValidFlowAddress)),
     defaultValues: {
       ...formData,
+      ...(IS_LOCAL_DEV
+        ? {
+            addr: '0x0ae53cb6e3f42a79',
+            name: 'FlowToken',
+            threshold: '1',
+            maxWeight: '2222',
+            publicPath: 'flowTokenBalance',
+          }
+        : undefined),
```

Comment:
added this logic to have some data to test easier, once reviewed will be removed. Use Token-weighted